astropy/modeling/powerlaws.py

```diff
@@ -129,6 +129,142 @@ def fit_deriv(x, amplitude, x_break, alpha_1, alpha_2):
         return [d_amplitude, d_x_break, d_alpha_1, d_alpha_2]
 
 
+class SmoothlyBrokenPowerLaw1D(Fittable1DModel):
+    """One dimensional smoothly broken power law model.
+
+    Parameters
+    ----------
+    amplitude : float
+        Model amplitude at the break point
+    x_break : float
+        Break point
+    alpha_1 : float
+        Power law index for x << x_break
+    alpha_2 : float
+        Power law index for x >> x_break
+    smooth : float
+        Smoothness parameter
+
+    See Also
+    --------
+    BrokenPowerLaw1D
+
+    Notes ----- Model formula (with :math:`A` for ``amplitude`` and
+    :math:`\\alpha_1` for ``alpha_1`` and :math:`\\alpha_2` for
+    ``alpha_2`` and :math:`C` for ``smooth`` and :math:`S` for the
+    sign of ``alpha_1-alpha_2``):
+
+        .. math::
+
+            f(x) = A (x / x_break) ^ {-\\alpha_1}
+                   \\left[
+                      \\frac{1 + (x / x_{break})^{|\\alpha_1-\\alpha_2| * C}}{2}
+                   \\right]}^{S / C}
+
+    """
+
+    amplitude = Parameter(default=1, bounds=[1.e-10, None])
```

Comment:
I agree.  However, how am I supposed to bound the parameter to positive values during fitting?
Setting bounds=[0, None] will not work since I need strictly positive values, not zero.